core/common/src/main/java/alluxio/wire/MasterWebUIInit.java

```diff
@@ -49,9 +49,9 @@ public boolean getDebug() {
   }
 
   /**
-   * Gets latest version.
+   * Gets newer version available.
    *
-   * @return the latest version
+   * @return the newer version available
```

Comment:
`true if a newer version is available, false otherwise`

Reply:
updated

Reply:
updated